runtime/onert/core/include/ir/OpSequence.h

```diff
@@ -75,13 +75,25 @@ class OpSequence
   std::vector<OperationIndex>::const_iterator begin() const { return _operations.begin(); }
   std::vector<OperationIndex>::const_iterator end() const { return _operations.end(); }
 
+public:
+  /**
+   * @brief Set @c true if any operation in this opSequence has dynamic input
+   *        or dynamic output;
+   *        @c false if all operations' inputs and outputs are static tensors
+   */
+  void hasDynamicTensor(bool hasDynamicTensor) { _has_dynamic_tensor = hasDynamicTensor; }
+  bool hasDynamicTensor() const { return _has_dynamic_tensor; }
```

Comment:
The name of Getter and Setter should be `has_dynamic_tensor` according to our coding convention.

Reply:
This will be fixed after getting answer for https://github.com/Samsung/ONE/pull/3042#discussion_r454075584.

Reply:
Fixed. Thanks.